test/e2e_node/eviction_test.go

```diff
@@ -286,51 +286,6 @@ var _ = SIGDescribe("LocalStorageSoftEvictionNotOverwriteTerminationGracePeriodS
 	})
 })
 
-// This test validates that in-memory EmptyDir's are evicted when the Kubelet does
-// not have Sized Memory Volumes enabled. When Sized volumes are enabled, it's
-// not possible to exhaust the quota.
-var _ = SIGDescribe("LocalStorageCapacityIsolationMemoryBackedVolumeEviction", framework.WithSlow(), framework.WithSerial(), framework.WithDisruptive(), feature.LocalStorageCapacityIsolation, nodefeature.Eviction, func() {
-	f := framework.NewDefaultFramework("localstorage-eviction-test")
-	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
-	evictionTestTimeout := 7 * time.Minute
-	ginkgo.Context(fmt.Sprintf(testContextFmt, "evictions due to pod local storage violations"), func() {
-		tempSetCurrentKubeletConfig(f, func(ctx context.Context, initialConfig *kubeletconfig.KubeletConfiguration) {
-			// setting a threshold to 0% disables; non-empty map overrides default value (necessary due to omitempty)
-			initialConfig.EvictionHard = map[string]string{string(evictionapi.SignalMemoryAvailable): "0%"}
-			if initialConfig.FeatureGates == nil {
-				initialConfig.FeatureGates = make(map[string]bool)
-			}
-			initialConfig.FeatureGates["SizeMemoryBackedVolumes"] = false
-		})
-
-		sizeLimit := resource.MustParse("100Mi")
-		useOverLimit := 200 /* Mb */
-		useUnderLimit := 80 /* Mb */
-		containerLimit := v1.ResourceList{v1.ResourceEphemeralStorage: sizeLimit}
-
-		runEvictionTest(f, evictionTestTimeout, noPressure, noStarvedResource, logDiskMetrics, []podEvictSpec{
-			{
-				evictionPriority: 1, // Should be evicted due to disk limit
-				pod: diskConsumingPod("emptydir-memory-over-volume-sizelimit", useOverLimit, &v1.VolumeSource{
-					EmptyDir: &v1.EmptyDirVolumeSource{Medium: "Memory", SizeLimit: &sizeLimit},
-				}, v1.ResourceRequirements{}),
-			},
-			{
-				evictionPriority: 0, // Should not be evicted, as container limits do not account for memory backed volumes
-				pod: diskConsumingPod("emptydir-memory-over-container-sizelimit", useOverLimit, &v1.VolumeSource{
-					EmptyDir: &v1.EmptyDirVolumeSource{Medium: "Memory"},
-				}, v1.ResourceRequirements{Limits: containerLimit}),
-			},
-			{
-				evictionPriority: 0,
-				pod: diskConsumingPod("emptydir-memory-innocent", useUnderLimit, &v1.VolumeSource{
-					EmptyDir: &v1.EmptyDirVolumeSource{Medium: "Memory", SizeLimit: &sizeLimit},
-				}, v1.ResourceRequirements{}),
-			},
-		})
-	})
-})
-
 // LocalStorageCapacityIsolationEviction tests that container and volume local storage limits are enforced through evictions
 var _ = SIGDescribe("LocalStorageCapacityIsolationEviction", framework.WithSlow(), framework.WithSerial(), framework.WithDisruptive(), feature.LocalStorageCapacityIsolation, nodefeature.Eviction, func() {
```

Comment:
Do yous still need the `feature.LocalStorageCapacityIsolation`? I think we can switch this to NodeConformance

Reply:
Maybe the feature but this test is in the eviction suite so it’s really long running and quite disruptive. I don’t think we should move this test into conformance.

I don’t think this is related to this PR though.

Reply:
We either need to explain what exactly is special about the test environment it needs to run in here: https://github.com/kubernetes/kubernetes/blob/2f7df335ad2d2ec8c3e876cbd2f4e5b3d3e17931/test/e2e/feature/feature.go#L208-L209 or remove the feature altogether.

It is already marked as Serial so it will not run with non-serial NodeConformance. Should it be running with eviction tests?

Reply:
You are commented on a feature that is not related to this KEP.

https://kubernetes.io/blog/2022/09/19/local-storage-capacity-isolation-ga/

This was already promoted to GA. I mainly changed this because it was using the feature gate.

> It is already marked as Serial so it will not run with non-serial NodeConformance. Should it be running with eviction tests?

Yes. It is using the eviction framework.